### ISSUE TITLE
FwdProxy AL2 Deployment Automation in Sandbox

### ISSUE THREAD

**Issue by npeterson54** [department-of-veterans-affairs/va.gov-team#81283]
## Description
Enable life traffic.  

## Resources 
Terraform needs created to replicate the behavior in Jenkins

## Acceptance Criteria
- [ ]  We have deployment parity with what we have in AL1

**Comment by ph-One** [comment]
Closing. Duplicate of https://github.com/department-of-veterans-affairs/va.gov-team/issues/87361

**Comment by ph-One** [comment]
This is being moved out in order to stabilize and monitor lower environments for a bit longer.

**Comment by ph-One** [comment]
Pushing this out two Sprints. We have uncovered a use case that was not previously accounted for, and additional work will have to be done before we can consider pushing the forward proxy beyond the lower environments.

**Comment by ph-One** [comment]
Pushing this out a sprint. https://github.com/department-of-veterans-affairs/vsp-platform-fwdproxy/issues/430 must be completed first

**Comment by stoiven** [comment]
in progress: verifying certs + configs match with legacy fwdproxy prior to the DNS cutoff (cc @ph-One) [we each will validate]

**Comment by stoiven** [comment]
FWDProxy config:

- `aft` exists in new fwdproxy, but not legacy

- `avs` has a different hostname and primary endpoint (https://github.com/department-of-veterans-affairs/vsp-platform-fwdproxy/pull/408) updated here
- `lgy_remittance` was a type `String`, changed it to be a `SecureString`  on param store
- `mvi` -> `/dsva-vagov/fwdproxy/sandbox/vetsgov-mvi.crt` AND `/dsva-vagov/fwdproxy/sandbox/vetsgov-mvi.key` were wrong, updated it to the same and matched the config code
- `res` has a slight difference with the added checks: https://github.com/department-of-veterans-affairs/vsp-platform-fwdproxy/pull/101/files
- `vet360 `-> /dsva-vagov/fwdproxy/sandbox/vet360.vets.gov.crt was a type string, changed it to be a SecureString  on param store
